docs/source/install.rst

```diff
@@ -175,17 +173,25 @@ If you are building from the git repository you have to first run::
 
 which will generate a configure script that can be used as described above.
 
-Before proceeding with the installation we need to add the datum grids. Unzip
-the contents of the *proj-datumgrid* package into ``data/``::
+Before proceeding with the installation we can add the data files.
+Unzip the contents of the *proj-data* package into ``data/``::
+
+    unzip proj-data-{PROJDATAVERSION}.zip -d proj-{PROJVERSION}/data/
 
-    unzip proj-datumgrid-1.7.zip -d proj-5.0.1/data/
+The installation will automatically move the data files to the correct location.
+Alternatively the the data files can be installed manually in the directory
```

Comment:
"the the" -> "the"

Reply:
I'm certain Kristian refers to Matt Johnson and his band here :-)

Reply:
No, but I recently got a new mechanical keyboard that i **really** like to type on, so I try to sneak in a few extra words now and then :-)

*clickety-clack*

Reply:
What is currently implemented in master and 7.0.0 will *not* install the TIFF grids in $prefix/share/proj if you unzip them in data. What is only supported is to unzip them directly in the $prefix/share/proj
This could be done of course (just adding a "*.tif" in the install-data-local target of data/Makefile.am), but I don't think it is necessary to correlate PROJ build and grid installation as we did before.

Reply:
Good point, Even. I was actually wondering if that was still the case. I'll update the text to reflect that. Perhaps the "official recommendation" should be to install the grids using `projsync` after a succesfull build and install?

Reply:
> Perhaps the "official recommendation" should be to install the grids using `projsync` after a succesfull build and install?

I've no strong preference between both options. The closest equivalent of unzipping to $prefix/share/proj would be "sudo projsync --system-directory" for Unix systems. Another difference is that with the unzipping method there's a clear versionning of what is installed, whereas with projsync that will be the state of the CDN at the time projsync is run.

Reply:
I think I'll just add the `--system-directory` call and then refer to the `projsync` doc page for further options that suit more specialized needs.